canonical-web-and-design/ubuntu.com
# Rebuild '/download/xilinx' page

## Done

**NOTE** to be released late on the 7th Dec

- Rebuild '/download/xilinx' page to add one more section and separate them by tabs as per [copydoc](https://docs.google.com/document/d/1DZxf-ZbJtyS69XQBslJry3Ien1FSHYLB2Qt78iRkhOs/edit#)
- Moves js for handling tabbed content into a static folder, to be included in '/openstack/install' and '/xlinx'


## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-10943.demos.haus/download/xilinx
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check the tabs work and that all links are correct
- Compare against copydoc

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4626


## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
